jackson-jr-test-module/pom.xml

```diff
@@ -0,0 +1,97 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <!-- This module was also published with a richer model, Gradle metadata,  -->
```

Comment:
We can leave this but we probably don't want to publish jar.

Reply:
understood. if its not required, then i will remove it.